src/api/resources/collect_record.py

```diff
@@ -179,6 +180,8 @@ def ingest(self, request, project_pk, *args, **kwargs):
         if "errors" in ingest_output:
             errors = ingest_output["errors"]
             return Response(errors, status=400)
+        elif "validate" in ingest_output:
+            return Response(ingest_output["validate"], status=400)
```

Comment:
Should probably make format of this and previous `errors` match

Reply:
@saanobhaai Wondering what that would look like?

Reply:
Sorry I meant the code, not the return vals, i.e. in one case we're setting a var (`errors`) and in the other we're not and using `ingest_output` directly.